spec/Notification-spec.ts

```diff
@@ -1,45 +1,46 @@
+import {expect} from 'chai';
 import * as Rx from '../dist/cjs/Rx';
-declare const expectObservable;
 
+declare const expectObservable;
 const Notification = Rx.Notification;
 
 /** @test {Notification} */
 describe('Notification', () => {
   it('should exist', () => {
-    expect(Notification).toBeDefined();
-    expect(typeof Notification).toBe('function');
+    expect(Notification).exist;
+    expect(Notification).to.be.a('function');
```

Comment:
Wow, I'm sorry you had to go through each of these assertions. Could have used something like https://github.com/crysalead-js/chai-jasmine. Did you try that?